BGL/doc/BGL/Concepts/FaceGraph.h

```diff
@@ -8,13 +8,23 @@ maintain faces described by halfedges, to provide access from a face to
 an incident halfedge, and to provide access from a halfedge to its incident
 face. 
 
+<h4>Associated Types:</h4>
+
+`boost::graph_traits<FaceGraph>::face_descriptor`
+A face descriptor corresponds to a unique face in an abstract graph instance.
+A face descriptor must be `DefaultConstructible`, `Assignable`, and `EqualityComparable`. 
```

Comment:
Some algorithm requires descriptors to be hashable. We should probably add the requirement here.

Reply:
How do you see that for vertex_descriptor. It is documented in the base concept in boost, and does not mention hashable. Shall we just repeat the vertex type in our documentation.

Reply:
Either we request it in functions that use it or we add it in `HalfedgeGraph` as you suggested.